src/OrbitClient.php

```diff
@@ -200,7 +200,7 @@ private function getRequestHeaders(array $options)
             $headers['X-Feature'] = 'akamai-idcta';
         }
         if ($this->options['env'] === 'stage') {
-            $headers['x-forge-environment'] = 'stage';
+            $headers['x-feature'] = 'forge-free';
```

Comment:
Can we remove the `x-forge-environment` header actually? Maybe a question for Tasio or Georgi.